README.md

```diff
@@ -48,6 +48,7 @@ The following OSINT resources have been implemented for each artifact type:
   - [AlienVault OTX](https://otx.alienvault.com/)
   - [ARIN](https://search.arin.net/rdap/)
   - [Bad Packets](https://badpackets.net/)
+  - [Criminalip](https://www.criminalip.io/)
```

Comment:
please place this in alphabetical order

Reply:
It's sorted alphabetically Please check it out. :)